tests/net/docker_io/mod.rs

```diff
@@ -62,3 +62,26 @@ fn test_dockerio_insecure() {
     let res = runtime.block_on(futcheck).unwrap();
     assert_eq!(res, true);
 }
+
+#[test]
+fn test_dockerio_unauthed() {
```

Comment:
_unauthed_ isn't subtle enough to describe on technical level in my understanding.

```suggestion
fn test_dockerio_anonymous_auth() {
```

Reply:
Yeah, it's a much better name